api/urls.py

```diff
@@ -9,4 +9,6 @@
     '',
     url(r'', include("api.v2.urls", namespace="default")),
     url(r'^v1/', include("api.v1.urls", namespace="v1")),
-    url(r'^v2/', include("api.v2.urls", namespace="v2")))
+    url(r'^v2/', include("api.v2.urls", namespace="v2")),
+    url(r'^status/', include("api.status.urls", namespace="status"))
+    )
```

Comment:
Again, Django 1.10 is going to remove `patterns( ... )` ([info](https://github.com/lenards/troposphere/commit/1e9c284ad6a875a2129b2f1b64dcfda2bd5df7ae))